src/main/java/com/dev/cinema/Main.java

```diff
@@ -42,13 +44,19 @@ public static void main(String[] args) throws AuthenticationException {
                 (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(movieSession);
         movieSessionService.findAvailableSessions(testMovie.getId(),
-                LocalDate.now()).forEach(System.out::println);
+                LocalDate.now()).forEach(log::info);
 
         AuthenticationService autService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
         autService.register("test@gmail.com", "qwerty");
+        log.info("Registration completed");
         User user1 = autService.login("test@gmail.com", "qwerty");
```

Comment:
put this in try-catch block like you did below

there should be no `throws AuthenticationException` in this method signature